src/writeData/subscriptions/components/BrokerFormContent.tsx

```diff
@@ -54,6 +54,7 @@ const BrokerFormContent: FC<Props> = ({
   const mqttProtocol = 'MQTT'
   const protocolList = [mqttProtocol]
   const [protocol, setProtocol] = useState(mqttProtocol)
+  const DEFAULT_PORT = formContent.authType === 'certificate' ? 8883 : 1883
```

Comment:
I think having the value change depending on the authentication selected might be confusing. What I proposed in the issue was actually having the value change depending on if the certificate feature flag is enabled. If certificate auth's feature flag is enabled, use 8883 as the default always. Otherwise use 1883 as the default always.

Reply:
ah gotcha. i thought as people use 1883 for non-secure connections and 8883 for secure ones, we should do this. just like 80 vs 443. But let me change this.

Reply:
I hear ya. I'm just worried since the people fill out the form from top to bottom they are going to make sure the port is correct and then move on to select their authentication. If toggling the authentication method changes the port I can see people not noticing the change and getting frustrated why the connection isn't working.